change/@fluentui-react-spinner-65e1fe2a-ca72-408e-90eb-6a1ca3f6caca.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30351&commitId=46bbdffd528c0db56faf555dd12283dafe84f566&env=prod&clientType=FLUENTUI)


  <details><summary><b>Spinner converged</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Spinner converged.Primary with Label - Dark Mode.chromium.png|287| Changed|
|Spinner converged.Inverted with Label - High Contrast.chromium.png|883| Changed|
|Spinner converged.Inverted with Label - Dark Mode.chromium.png|467| Changed|
|Spinner converged.Inverted - High Contrast.chromium.png|221| Changed|
|Spinner converged.Primary - Dark Mode.chromium.png|72| Changed|
|Spinner converged.Inverted.chromium.png|117| Changed|
|Spinner converged.Primary + size.chromium.png|767| Changed|
|Spinner converged.Inverted - Dark Mode.chromium.png|117| Changed|
|Spinner converged.Primary with Label - High Contrast.chromium.png|868| Changed|
|Spinner converged.Primary with Label - RTL.chromium.png|426| Changed|
|Spinner converged.Inverted with Label.chromium.png|467| Changed|
|Spinner converged.Primary.chromium.png|106| Changed|
|Spinner converged.Primary - High Contrast.chromium.png|217| Changed|
|Spinner converged.Primary with Label.chromium.png|426| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>